docs/pages/concepts/offloading/mutating-webhook.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Mutating webhook
+weight: 5
+---
+
+This section explains how the Liqo Webhook mutates Pods created within a Liqo-enabled namespace, enabling them to be possibly offloaded to remote clusters and enforcing user-specified constraints.
+
+As detailed in the [namespace offloading](/usage/namespace_offloading) section, the creation of a *NamespaceOffloading* resource inside a given namespace extends that namespace to remote clusters (possibly remapped, to prevent naming conflicts).
```

Comment:
I would add something like:
The *NamespaceOffloading* resource is created on the local cluster.
(although the name "local" looks confusing to me, I would prefer the term "home", but anyway, this has to be coherent with the rest of the documentation, so feel free to use whatever looks better for you)